app/api/dao/user.py

```diff
@@ -154,22 +156,26 @@ def list_users(user_id, is_verified=None):
         else:
             list_of_users = [user.json() for user in users_list]
 
+        for user in list_of_users:
+            relation = MentorshipRelationDAO.list_current_mentorship_relation(user['id'])
```

Comment:
Make a separate method called isUserInRelation(user_id)

Reply:
Ok I got that.